components/home/Home.js

```diff
@@ -210,6 +210,12 @@ class Home extends Component {
                     </Section> : null
                     }
                 </ScrollView>
+                <VStack alignItems="center" mb={8}>
+                    <TouchableOpacity onPress={() => Linking.openURL('https://forms.gle/aXVjxVrQU6jm3KUx6')}><Text style={{ color: 'blue' }}>Submit Feedback</Text></TouchableOpacity>
+                    <Text textAlign="center">
+                      {"Built by students at Olin College of Engineering in partnership with the Volpe Center and Santos Family Foundation"}
```

Comment:
Seems like a lot to have this on every single screen

Reply:
Roger that, I'll remove it from most of them

Reply:
Definitely leave it on the Welcome Screen. Maybe include it on the Final Report screen